airbyte-server/src/main/java/io/airbyte/server/converters/SchemaConverter.java

```diff
@@ -127,4 +136,68 @@ private static SourceSchemaField toApiField(Field persistenceField) {
     return field;
   }
 
+  public static ConfiguredAirbyteCatalog toApi(AirbyteCatalog1 catalog) {
```

Comment:
Was the method misnamed `toApi`?

- Input AirbyteCatalog1 is `io.airbyte.api.model.AirbyteCatalog1`
- Output ConfiguredAirbyteCatalog is `io.airbyte.protocol.models.ConfiguredAirbyteCatalog`

Should this be called something like `toProtocol` (or `toPersistence` to keep with the previous vocabulary) instead?

Reply:
yeah. definitely feel free to change the name. i was just writing this code to sanity check what the conversions look like. i wasn't being very thoughtful about the names